client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -56,6 +56,21 @@
             :group-select="false" />
         </action-stepper-action>
 
+        <!-- Assign place/status -->
```

Comment:
@sakutademos the new select is now played between the "Add Tags" and "Delete Tags" sections. I meant to place it above the "Add Tags" section.